### ISSUE TITLE
Cap the number of tags visible on an extension page

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vsmarketplace#1]
**Describe the bug**

There should be some maximum number of tags visible (10-20?), with a ... or "load more" button to show them all

**To Reproduce**

Check out https://marketplace.visualstudio.com/items?itemName=jabacchetta.vscode-essentials

**Screenshots**

![image](https://user-images.githubusercontent.com/2193314/108708519-67473a80-74c6-11eb-8f67-696a4014ca4d.png)

**Comment by RandomFractals** [comment]
I disagree. Limiting tags to 10 or 20 might cut ways to find extensions that cover many data files or other features. I have about 50 in mine, and not sure how I would trim them to 10: https://marketplace.visualstudio.com/items?itemName=RandomFractalsInc.vscode-data-preview

**Comment by Tyriar** [comment]
@RandomFractals I'm not saying they should impact search or discoverability in any way, just they should not all show on the extension detail page without clicking to expand since it can be a lot and it pushed important UI elements down.

**Comment by RandomFractals** [comment]
@Tyriar I like the idea of maybe listing 10 or 20 and then adding `...` for expending to see them all.
